### DISCUSSION TITLE
MISP integration

### DISCUSSION THREAD

**Discussion by ClioSecurity** [SecurityBrewery/catalyst#1100] [Ideas]
Hello Team,

We are exploring alternatives to The Hive (from StrangeBee) and came across your impressive project.

We found some unofficial instructions about integrations with Wazuh. Is there any way to integrate Catalyst with MISP as well?

It would be fantastic if this integration were possible. If it isn’t currently available, I encourage you to consider it, as such a feature could elevate your project to very valid alternative.

Thank you all.

**Comment by cugu** [comment]
@ClioSecurity What features would you expect from a MISP integration?

**Reply by ClioSecurity** [reply]
Hello Cugu,

Similar to how MISP integrates with The Hive, we would expect Catalyst to be able to utilize international databases of IOCs and known malware that MISP offers. This is very different from databases like VirusTotal, for example.

We believe that with this integration, Catalyst could significantly improve the efficiency and speed of analysts' work.

I would suggest reaching out directly to CIRCL, for instance, and requesting access to their database. They can create an account for you, allowing you to explore the possibilities firsthand. It's free of charge, so you can see the potential yourself.

Best regards,
